pkg/subctl/cmd/benchmark.go

```diff
@@ -18,30 +20,38 @@ var (
 		Short: "Benchmark throughput between two clusters",
 		Long:  "This command runs throughput tests between two clusters",
 		Args: func(cmd *cobra.Command, args []string) error {
-			return checkThroughputArguments(args)
+			return checkThroughputArguments(args, intraCluster)
 		},
 		Run: testThroughput,
 	}
 	benchmarkLatenchCmd = &cobra.Command{
-		Use:   "latency <kubeconfig1> <kubeconfig2>",
+		Use:   "latency <kubeconfig1> [<kubeconfig2>]",
 		Short: "Benchmark latency between two clusters",
 		Long:  "This command runs latency benchmark tests between two clusters",
 		Args: func(cmd *cobra.Command, args []string) error {
-			return checkThroughputArguments(args)
+			return checkThroughputArguments(args, intraCluster)
 		},
 		Run: testLatency,
 	}
 )
 
 func init() {
+	benchmarkLatenchCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
+		"Run benchmark latency test with intra-cluster pods")
+
+	benchmarkThroughputCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
+		"Run benchmark latency test with intra-cluster pods")
+
```

Comment:
I’m wondering if we need a separate flag for this, or if we can use the number of kubeconfigs given to choose... What do you think?

Reply:
Having the flag is more explicit.
Without the flag, if a user forget to add second kubeconfig the result will be another type of test.

Reply:
Right, more explicit is better since the failure scenario isn’t obvious.